generic_io/rl6/firmware.lua

```diff
@@ -0,0 +1,74 @@
+local CHANNELS_NUMBER = 6
+
+function main()
+  scheduler.add(1000, send_telemetry)
+
+  enapter.register_command_handler('close_channel', command_close_channel)
+  enapter.register_command_handler('close_all_channels', command_close_all_channels)
+  enapter.register_command_handler('open_channel', command_open_channel)
+  enapter.register_command_handler('open_all_channels', command_open_all_channels)
+  enapter.register_command_handler('is_channel_closed', command_is_channel_closed)
+  enapter.register_command_handler('all_channels_state', command_all_channels_state)
+end
+
+function send_telemetry()
+  local telemetry = {
+    status = 'ok',
+    alerts = {}
+  }
+
+  for i=1,CHANNELS_NUMBER,1 do
+    local val, res = rl6.get(i)
+    if res and res ~= 0 then
+      enapter.log(rl6.err_to_str(res))
+      telemetry.status = 'error'
+      telemetry.alerts = {'cannot_read_channel_state'}
+    end
+    telemetry['channel_'..tostring(i)..'_closed'] = val
+  end
+
+  enapter.send_telemetry(telemetry)
+end
+
+function command_close_channel(ctx, args)
+  if not args.channel or type(args.channel) ~= 'number' then
+    ctx.error('channel arg is required')
```

Comment:
Error messages are different here in `command_open_channel` and `command_is_channel_closed`. Suppose "channel arg is required and must be a number" is the best choice.